controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -321,6 +344,10 @@ func (r *StorageClassClaimReconciler) reconcileConsumerPhases() (reconcile.Resul
 				} else if resource.Name == "ceph-rbd" {
 					storageClass = r.getCephRBDStorageClass(data)
 				}
+				if storageClass.Annotations == nil {
+					storageClass.Annotations = map[string]string{}
+				}
+				storageClass.Annotations[StorageClassClaimNamespaceAnnotation] = r.storageClassClaim.Namespace
```

Comment:
Please use Get/SetAnnotations to read/update the annotations:
```suggestion
				if annotations := storageClass.GetAnnotations(); annotations == nil {
					storageClass.SetAnnotations(map[string]string{
						StorageClassClaimNamespaceAnnotation: r.storageClassClaim.Namespace
					})
				} else {
					annotations[StorageClassClaimNamespaceAnnotation] = r.storageClassClaim.Namespace
				}
```